lib/pages/add_bridge/add_bridge.dart

```
@@ -391,6 +523,45 @@ class BotController extends State<AddBridge> {
   }
 
   /// Disconnect from a social network
+  Future<void> disconnectBridgeApi(
+      BuildContext context,
+      SocialNetwork network,
+      ConnectionStateModel connectionState,
+      {String loginId = 'all'}
+      ) async {
+    final bridgePathIdentifier  = getBridgePath(network);
+    final userId = client.userID;
+
+    final logoutUrl = 'https://matrix.$serverUrl/_matrix/$bridgePathIdentifier/provision/v3/logout/$loginId?user_id=$userId';
+
+    Future.microtask(() {
+      connectionState.updateConnectionTitle(L10n.of(context)!.loadingDisconnectionDemand);
+    });
+
+    try {
+      final response = await requestPost(logoutUrl);
+
+      if (response.statusCode == 200) {
+        if (kDebugMode) {
+          print("Successful disconnection for ${network.name}");
+        }
+        setState(() => network.updateConnectionResult(false));
+      } else {
+        if (kDebugMode) {
+          print("Disconnection error: ${response.statusCode}");
+        }
```

This error is not propagated or shown to user